spikeinterface/comparison/groundtruthstudy.py

```diff
@@ -203,28 +199,39 @@ def aggregate_dataframes(self, copy_into_folder=True, **karg_thresh):
 
         return dataframes
 
-    def compute_metrics(self, rec_name, metric_names=['snr'],
+    def compute_metrics(self, rec_name, sorter_name=None, metric_names=['snr'],
                 ms_before=3., ms_after=4., max_spikes_per_unit=500, 
-                n_jobs=-1, total_memory='1G', **snr_kargs):
+                n_jobs=-1, total_memory='1G', templates_only=False, **snr_kargs):
 
         rec = self.get_recording(rec_name)
-        gt_sorting = self.get_ground_truth(rec_name)
-        
+        if sorter_name is None:
+            sorting = self.get_ground_truth(rec_name)
+            sorter_name = 'gt'
+        else:
+            assert sorter_name in self.sorter_names
+            sorting = self.get_sorting(sorter_name, rec_name)
+
```

Comment:
Here I would prefer something super explicit.
This sorter_name=None is not enought explicit.
For me at the study level metrics are GT based.
So if it make sens in some situation to have metrics on sorting ouput I would prefer an other method for this.
Even if internally the implemtation is like this.